External/FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -5439,11 +5655,17 @@ void InstallOpcodeHandlers(Context::OperatingMode Mode) {
 
     {0x20, 6, &OpDispatchBuilder::ALUOp},
 
+    {0x27, 1, &OpDispatchBuilder::DAAOp},
```

Comment:
These instructions are only valid in 32-bit mode, so these should be moved down to the array below called `BaseOpTable_32`